src/autocompleters/sampleAutocompleter.ts

```diff
@@ -11,7 +11,9 @@ class CountryAutocompleter extends Autocompleter {
 		const country = ["Finland", "Sweden", "Norway"];
 		// for example return all options which start with the user input
 		await interaction.respond(
-			country.filter((c) => c.startsWith(interaction.options.getFocused() as string)).map((c) => ({ name: c, value: c })),
+			country
+				.filter((c) => c.startsWith(interaction.options.getFocused() as string))
```

Comment:
Might want to make this check case insensitve

Reply:
thats just the sample file I did that with my country autocompletion

Reply:
but you can add it to the sample file as well if you want